unit tests fail often due to unsorted lists

Tracking this because I'm sick of seeing red travis runs for this simple issue:

```
System check identified no issues (0 silenced).

cd rootfs \

		&& coverage run manage.py test --settings=api.settings.testing --noinput registry api scheduler.tests \

		&& coverage report -m

Creating test database for alias 'default'...

.............................................................F..............................................................................................................................................................................................................................................................................

======================================================================

FAIL: test_manage_idn_domain (api.tests.deployments.test_domain.DomainTest)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/deis/controller/rootfs/api/tests/deployments/test_domain.py", line 114, in test_manage_idn_domain

    self.assertEqual([self.app_id, ace_domain], expected, msg)

AssertionError: Lists differ: ['candid-organism', 'xn--80ahd1agd.ru'] != ['xn--80ahd1agd.ru', 'candid-organism']

First differing element 0:

candid-organism

xn--80ahd1agd.ru

- ['candid-organism', 'xn--80ahd1agd.ru']

+ ['xn--80ahd1agd.ru', 'candid-organism'] : failed on 'домена.ru'

----------------------------------------------------------------------

Ran 332 tests in 605.528s

FAILED (failures=1)

Destroying test database for alias 'default'...

make: *** [test-unit] Error 1
```

--- a/rootfs/api/tests/deployments/test_domain.py
+++ b/rootfs/api/tests/deployments/test_domain.py
@@ -111,7 +111,7 @@ class DomainTest(APITestCase):
             url = '/v2/apps/{app_id}/domains'.format(app_id=self.app_id)
             response = self.client.get(url)
             expected = [data['domain'] for data in response.data['results']]
-            self.assertEqual([self.app_id, ace_domain], expected, msg)
+            self.assertEqual(sorted([self.app_id, ace_domain]), sorted(expected), msg)
 
             # Verify creation failure for same domain with different encoding
             if ace_domain != domain:
@@ -148,7 +148,7 @@ class DomainTest(APITestCase):
                 url = '/v2/apps/{app_id}/domains'.format(app_id=self.app_id)
                 response = self.client.get(url)
                 expected = [data['domain'] for data in response.data['results']]
-                self.assertEqual([self.app_id, ace_domain], expected, msg)
+                self.assertEqual(sorted([self.app_id, ace_domain]), sorted(expected), msg)
 
                 # Delete
                 url = '/v2/apps/{app_id}/domains/{hostname}'.format(hostname=ace_domain,
@@ -175,7 +175,7 @@ class DomainTest(APITestCase):
                 url = '/v2/apps/{app_id}/domains'.format(app_id=self.app_id)
                 response = self.client.get(url)
                 expected = [data['domain'] for data in response.data['results']]
-                self.assertEqual([self.app_id, ace_domain], expected, msg)
+                self.assertEqual(sorted([self.app_id, ace_domain]), sorted(expected), msg)
 
                 # Delete
                 url = '/v2/apps/{app_id}/domains/{hostname}'.format(hostname=unicode_domain,
@@ -220,7 +220,7 @@ class DomainTest(APITestCase):
             url = '/v2/apps/{app_id}/domains'.format(app_id=self.app_id)
             response = self.client.get(url)
             expected = [data['domain'] for data in response.data['results']]
-            self.assertEqual([self.app_id, domain], expected, msg)
+            self.assertEqual(sorted([self.app_id, domain]), sorted(expected), msg)
 
             # Delete
             url = '/v2/apps/{app_id}/domains/{hostname}'.format(hostname=domain,

--- a/rootfs/api/tests/test_domain.py
+++ b/rootfs/api/tests/test_domain.py
@@ -109,7 +109,7 @@ class DomainTest(APITestCase):
             url = '/v2/apps/{app_id}/domains'.format(app_id=self.app_id)
             response = self.client.get(url)
             expected = [data['domain'] for data in response.data['results']]
-            self.assertEqual([self.app_id, ace_domain], expected, msg)
+            self.assertEqual(sorted([self.app_id, ace_domain]), sorted(expected), msg)
 
             # Verify creation failure for same domain with different encoding
             if ace_domain != domain:
@@ -146,7 +146,7 @@ class DomainTest(APITestCase):
                 url = '/v2/apps/{app_id}/domains'.format(app_id=self.app_id)
                 response = self.client.get(url)
                 expected = [data['domain'] for data in response.data['results']]
-                self.assertEqual([self.app_id, ace_domain], expected, msg)
+                self.assertEqual(sorted([self.app_id, ace_domain]), sorted(expected), msg)
 
                 # Delete
                 url = '/v2/apps/{app_id}/domains/{hostname}'.format(hostname=ace_domain,
@@ -173,7 +173,7 @@ class DomainTest(APITestCase):
                 url = '/v2/apps/{app_id}/domains'.format(app_id=self.app_id)
                 response = self.client.get(url)
                 expected = [data['domain'] for data in response.data['results']]
-                self.assertEqual([self.app_id, ace_domain], expected, msg)
+                self.assertEqual(sorted([self.app_id, ace_domain]), sorted(expected), msg)
 
                 # Delete
                 url = '/v2/apps/{app_id}/domains/{hostname}'.format(hostname=unicode_domain,
@@ -218,7 +218,7 @@ class DomainTest(APITestCase):
             url = '/v2/apps/{app_id}/domains'.format(app_id=self.app_id)
             response = self.client.get(url)
             expected = [data['domain'] for data in response.data['results']]
-            self.assertEqual([self.app_id, domain], expected, msg)
+            self.assertEqual(sorted([self.app_id, domain]), sorted(expected), msg)
 
             # Delete
             url = '/v2/apps/{app_id}/domains/{hostname}'.format(hostname=domain,

